README.md

```diff
@@ -221,13 +221,13 @@ To develop NPD on Windows you'll need to setup your Windows machine for Go devel
 ```powershell
 # Run these commands in the node-problem-detector directory.
 
-# Build
+# Build in MINGW64 Window
 make clean windows-binaries
 
-# Test
+# Test in MINGW64 Window
 make test
 
-# Run with containerd log monitoring enabled. (Assumes containerd is installed.)
+# Run with containerd log monitoring enabled in Powershell/Command Prompt. (Assumes containerd is installed.)
```

Comment:
`%CD%` won't work in Powershell. Perhaps we should make the command exclusive to MinGW command prompt. In that case you need `$PWD`.

Reply:
Also the `sc.exe` can only run in an Administrator mode prompt.